mdn_yari/explores/page_table.explore.lkml

```diff
@@ -0,0 +1,14 @@
+include: "//looker-hub/mdn_yari/views/page_table.view.lkml"
+
+explore: page_table {
```

Comment:
Is there anything in this explore that the `page` explore doesn't have?

Reply:
yes, Florian from MDN was trying to use Looker and noticed the following was missing from the `page` Explore:
https://github.com/mozilla/looker-hub/blob/main/mdn_yari/views/page_table.view.lkml#L335-L351

I assume this is not the first time this happens, I'm not quite sure what is our standard approach to handle this kind of case. In the PR changes, there are two different approaches suggested (creating a new Explore for Page Table or extending the page view to include those dimensions).

Once we agree on how to proceed I will remove the unnecessary changes. I guess one more approach would be to join the two views together in the existing `page` Explore.

Is this something that happens often, and how should cases like this be resolved @scholtzan ?

Reply:
This is the first time this has happened. So for now this is fine until we come across some other cases where this happens which is when we can think of a more general solution.